bootstrap

```diff
@@ -112,8 +113,10 @@ extract(Binary) ->
     {ok, Contents}.
 
 request(Url) ->
+    HttpOptions = [{relaxed, true} | get_proxy_auth()],
```

Comment:
Yeah, I'm not sure about that. We could probably cache it by putting it in the ENV (for regular rebar3) or the process dictionary (for bootstrap?).